sim/firesim-lib/src/main/scala/endpoints/SimpleNICWidget.scala

```diff
@@ -190,9 +190,13 @@ class SimpleNICWidget(implicit p: Parameters) extends EndpointWidget()(p)
   val NICtokenToBig = Module(new NICTokenToBigTokenAdapter)
 
   val target = io.hPort.hBits
-  val tFire = io.hPort.toHost.hValid && io.hPort.fromHost.hReady && io.tReset.valid
-  val targetReset = tFire & io.tReset.bits
-  io.tReset.ready := tFire
+  val fixMeOnNextRocketBump = true.B // Need a dummy predicate to exclude for now
```

Comment:
Why would a Rocket bump resolve this?

Reply:
DecoupledHelper.fire() can take no arguments to take the conjunction of all booleans provided to it.